Tests/scripts/infrastructure_tests/test_collect_tests.py

```diff
@@ -259,155 +261,158 @@ def test_nightly(monkeypatch, case_mocker: CollectTestsMocker, collector_class:
     'case_mocker,expected_tests,expected_packs,expected_machines,expected_modeling_rules_to_test,'
     'collector_class_args,mocked_changed_files,mocked_packs_files_were_moved_from,expected_packs_to_upload',
     (
-        # (0) change in a sanity-collection-triggering file, expecting xsoar sanity tests to be collected
```

Comment:
the indentation here will affect git-blame, if it's the only change around, consider reverting it